workflows/python/sdk/order-processor/app.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import threading
+from time import sleep
+from dapr.ext.workflow import WorkflowRuntime
+from workflow import order_processing_workflow, notify_activity, process_payment_activity, \
+    verify_inventory_activity, update_inventory_activity, requst_approval_activity
+from dapr.clients import DaprClient
+from model import InventoryItem, OrderPayload
+from datetime import datetime
+from dapr.conf import settings
+
+store_name = "statestore-actors"
+workflow_component = "dapr"
+workflow_name = "order_processing_workflow"
+default_item_name = "cars"
+
+class WorkflowConsoleApp:    
+    def main(self):
+        print("*** Welcome to the Dapr Workflow console app sample!")
+        print("*** Using this app, you can place orders that start workflows.")
+        # Wait for the sidecar to become available
+        sleep(5)
+
+        workflowRuntime = WorkflowRuntime(settings.DAPR_RUNTIME_HOST, settings.DAPR_GRPC_PORT)
+        workflowRuntime.register_workflow(order_processing_workflow)
+        workflowRuntime.register_activity(notify_activity)
+        workflowRuntime.register_activity(requst_approval_activity)
+        workflowRuntime.register_activity(verify_inventory_activity)
+        workflowRuntime.register_activity(process_payment_activity)
+        workflowRuntime.register_activity(update_inventory_activity)
+        workflowRuntime.start()
+
+        daprClient = DaprClient(address=f'{settings.DAPR_RUNTIME_HOST}:{settings.DAPR_GRPC_PORT}')
+        baseInventory = {}
+        baseInventory["paperclip"] = InventoryItem("Paperclip", 5, 100)
+        baseInventory["cars"] = InventoryItem("Cars", 15000, 100)
+        baseInventory["computers"] = InventoryItem("Computers", 500, 100)
+
+        self.restock_inventory(daprClient, baseInventory)
+
+        print("==========Begin the purchase of item:==========")        
+        item_name = default_item_name
+        order_quantity = 11
+
+        total_cost = int(order_quantity) * baseInventory[item_name].per_item_cost
+        order = OrderPayload(item_name=item_name, quantity=int(order_quantity), total_cost=total_cost)
+        print(f'Starting order workflow, purchasing {order_quantity} of {item_name}')
+        start_resp = daprClient.start_workflow(workflow_component=workflow_component,
+                                               workflow_name=workflow_name,
+                                               input=order)
+        _id = start_resp.instance_id
+
+        def prompt_for_approval(daprClient: DaprClient):
+            """This is a helper function to prompt for approval.
+            Not using the prompt here ACTUALLY, as quickstart validation is required to be automated.
```

Comment:
thank you i like the way you handle this.

In the future we can make this an optional part of the doc too.

Reply:
@hhunter-ms